### ISSUE TITLE
Requirements could not be resolved

### ISSUE THREAD

**Issue by fullkawa** [a1comms/GaeSupportLaravel#38]
When I did `composer update`, I got a error message.

```
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - a1comms/gae-support-laravel dev-php72-laravel55 requires tomwalder/php-gds dev-grpc -> no matching package found.
    - a1comms/gae-support-laravel dev-php72-laravel55 requires tomwalder/php-gds dev-grpc -> no matching package found.
    - Installation request for a1comms/gae-support-laravel dev-php72-laravel55 -> satisfiable by a1comms/gae-support-laravel[dev-php72-laravel55].
```
tomwalder/php-gds has not "dev-grpc" version now.
https://packagist.org/packages/tomwalder/php-gds

**Comment by iamacarpet** [comment]
Hello @fullkawa ,

Apologies for the trouble; we don't have a stable version yet while the runtime is still in BETA, so we're still making breaking changes, but I'll try and post updates if I know you are depending on it.

This is part of our work to switch Datastore access from REST to gRPC for a speed improvement.

Google haven't merged it into `google-cloud-php` yet, which is stopping us merging into tomwalder/php-gds

In our version, we had to manually include some repository links in `composer.json` to look at our forks instead:

```json
    "repositories": [
        {
          "type": "git",
          "url": "https://github.com/a1comms/GaeSupportLaravel.git"
        },
        {
            "type": "git",
            "url": "https://github.com/a1comms/php-gds.git"
        },
        {
            "type": "git",
            "url": "https://github.com/a1comms/datastore-grpc.git"
        }
    ]
```

And in the `"require"` block, you may have to include these too:

```json
        "google/datastore": "dev-master",
        "tomwalder/php-gds": "dev-grpc",
```

We are looking to get the changes merged upstream so this isn't required any more and will update as soon as we can.

**Comment by fullkawa** [comment]
Hello @iamacarpet,
I appreciate your response.
I will develop some services on GAE+Laravel 5.5(or later). So I hope someday this problem will be solved fundamentally.
Your comment is enough for me at the moment.

**Comment by iamacarpet** [comment]
Hello @fullkawa ,

`google-cloud-php` have merged and released their changes for grpc support in Datastore, but we still don't have it available in `php-gds` yet, as we're just waiting for @tomwalder to review the PR on his repo.

In the meantime, an update on what I said before, you'll need:

```json
    "repositories": [
        {
          "type": "git",
          "url": "https://github.com/a1comms/GaeSupportLaravel.git"
        },
        {
            "type": "git",
            "url": "https://github.com/a1comms/php-gds.git"
        }
     ]
```

And in the `"require"` block, you may have to include these too:

```json
    "tomwalder/php-gds": "dev-grpc",
```

**Comment by iamacarpet** [comment]
Hello @fullkawa ,

Sorry for the late update; you may have already noticed, we forked our own `php-gds` and registered it with packagist, as we didn't think @tomwalder was around at the moment to accept the PR.

You shouldn't need any of these fixes any more, it should just work.

We've also released a stable version in the 5.5.x branch now, so we'll try not to break things going forward.

Regards,
iamacarpet
